packages/table/README.md

```diff
@@ -2,7 +2,28 @@
 
 ![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/table.svg)
 
-## Installation
+> _Upgrading from pre-v10 to v11? Check out our [upgrade guide](https://github.com/mongodb/leafygreen-ui/blob/main/packages/table/UPGRADE.md)._
+
+The Table component displays data in rows and columns and optionally supports row selection, sorting, and other features.
+
+## Supported features
+
+- Virtualized scrolling
+- Nested rows
+- Expandable rows
+- Selectable rows
+- Sortable rows
+- Sticky headers
+
+While other features from [`react-table`](https://github.com/tanstack/table#quick-features) could be supported, we discourage developers from utilizing them unless they are explicitly supported by Leafygreen, as they would not align with MongoDB design systems guidelines.
```

Comment:
I think we should rm this -- leaving it here encourages people to check out the docs and hack around our implementation